test/integration/asset_access_options_integration_test.rb

```diff
@@ -82,7 +82,7 @@ class AssetAccessOptionsIntegrationTest < ActionDispatch::IntegrationTest
           it "marks replacement attachment as access limited in Asset Manager" do
             Services.asset_manager.expects(:create_whitehall_asset).with do |params|
               params[:legacy_url_path] =~ /big-cheese/ &&
-                params[:access_limited] == %w[user-uid] &&
+                params[:access_limited] == [organisation.content_id] &&
```

Comment:
Shouldn't the key now be `access_limited_organisation_ids` instead of `access_limited`? 🤔 

At least, that's according to [AttachmentUpdater.get_asset_attributes](https://github.com/alphagov/whitehall/blob/9b9f24b46530179f4eb6113c491181829cad1480/app/services/asset_manager/attachment_updater.rb#L50).

Throughout this file we have assertions that `access_limited` contains organisation IDs, but I thought that could only be used to represent User IDs rather than Organisation IDs.

Reply:
Quite right @ollietreend, I have updated those assertions what led me to changes in `app/workers/asset_manager_create_asset_worker.rb `and  `app/workers/asset_manager_create_whitehall_asset_worker.rb` and a bunch of related tests.